### PR TITLE
Feature / Add price calculator to product page

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Remove test price page.
- Add some additional properties to product page props.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #303** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/303" target="_blank">Graphite</a> 
    * **PR #305** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/305" target="_blank">Graphite</a> 
      * **PR #306** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/306" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/306?utm_source=stack-comment).

File: apps/store/src/pages/products/[product].tsx [hunk 1]
```diff
@@ -26,11 +28,30 @@ export const getServerSideProps: GetServerSideProps<ProductPageProps> = async (c
...
+
+    const lineItem = priceIntent.lines?.[0]
+
+    return {
+      props: {
+        cmsProduct,
+        product: {
+          ...product,
+          price: lineItem?.price.amount ?? null,
+          currencyCode: CurrencyCode.Sek,
```
Comment (guilhermespopolin) [inline apps/store/src/pages/products/[product].tsx:46]: I guess it should be `lineItem?.price.currencyCode` right?! One issue tho is that `currencyCode` is required on `ProductPageProps` 🤔 . Maybe `lineItem?.price.currencyCode ?? CurrencyCode.Sek` or any other value we should consider the default one...

**Review by guilhermespopolin — APPROVED** [review]
image

File: apps/store/src/pages/products/[product].tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/pages/products/[product].tsx:46]: Ahh yes, that's a placeholder.

Don't really know where it should come from tho. I would have to be available before we have a price tho. And I'm not sure if I like that we infer it from the locale but maybe that is how we should do it rather.

Comment (robinandeer) [reply apps/store/src/pages/products/[product].tsx:46]: I def. don't want to have a "default" currency code tho 😅 there's no default market/country in the coverage store -- they are all equal hehe

Let's not make that mistake again

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/306?utm_source=stack-comment).


Job Reference: bDmM3f7qDjlaCMyAuyQY

**Pull request merged**